### ISSUE TITLE
Needed changes in light of the creation of v2

### ISSUE THREAD

**Issue by jessp01** [solworktech/mdtopdf#45]
Hi @mandolyte ,

Since we're now on v2, some structure changes are required as, when one runs:
```sh
go install github.com/mandolyte/mdtopdf/cmd/md2pdf@latest
```
The received version is v1.5.3, not v2.2.1.

When trying to explicitly install v2.2.1:
```sh
$ go install github.com/mandolyte/mdtopdf/cmd/md2pdf@v2.2.1
```

One would get this message:
> go: github.com/mandolyte/mdtopdf/cmd/md2pdf@v2.2.1: github.com/mandolyte/mdtopdf@v2.2.1: invalid version: module contains a go.mod file, so module path must match major version ("github.com/mandolyte/mdtopdf/v2")

Please see https://go.dev/blog/v2-go-modules

Happy to help with the restructure if you'd like.

Cheers,

**Comment by mandolyte** [comment]
@jessp01 I have a counter proposal... while I love to code and this and other projects were done after I retired, I don't quite see myself spending as much time coding going forward. I also just finished a 3 year stint of working for a non-profit doing web apps with JavaScript. And now my kids have move to Texas and England. I don't think I'll be doing much coding even tho I enjoy it. 

So my counter proposal is this: would you be willing to take over this package? Thanks for your interest and energy!

**Comment by mandolyte** [comment]
@jessp01 I think I can just transfer the repo to you. But thought I'd make sure that was the best approach. What do you recommend?

**Comment by jessp01** [comment]
Hi @mandolyte ,

Yes, I think that'd be fine:)

**Comment by mandolyte** [comment]
I got an error. Looks like you need to (re)move your repo with same name...
![image](https://github.com/mandolyte/mdtopdf/assets/1289437/3545984e-6300-4518-a81b-601f8aa9693b)

**Comment by jessp01** [comment]
Ah, I thought you'd just grant my user write privileges on the existing repo, or in other words, set me up as a "collaborator":)
https://github.com/mandolyte/mdtopdf/settings/access

That way, you could still co-maintain it with me whenever you're so inclined and we needn't redirect from https://github.com/mandolyte/mdtopdf.

**Comment by mandolyte** [comment]
I see. That makes sense. I have added you. You should have an invite.

**Comment by jessp01** [comment]
Happily accepted and just merged my own pull :)
Thanks, @mandolyte. Enjoy your retirement!

**Comment by mandolyte** [comment]
@jessp01 I noticed some new issues. Will you be able to take a look? BTW, I am in England also until April (near London). Warmer, but wetter than Cincinnati :-)

**Comment by jessp01** [comment]
Hi @mandolyte ,

Hope you're well. I can only see two issues from the same user pertaining to the use of the Noto Sans font. I've posted a comment on these. Were you referring to those or something else?

PS
Enjoy your time in England:)

**Comment by mandolyte** [comment]
Yes, those were the ones. Looks like he isn't interested now. Consider
closing unless you can replicate (or if you can't replicate, demonstrate
what should have worked).

Thanks
